bafbooks.sql

```diff
@@ -0,0 +1,109 @@
+DROP DATABASE bafbooks;
```

Comment:
While this can be helpful in initial setup, migrations do this work, you should have a SQL file in your repo.